EKF/ekf_helper.cpp

```diff
@@ -767,28 +767,28 @@ bool Ekf::resetMagHeading(Vector3f &mag_init, bool increase_yaw_var, bool update
 }
 
 // Calculate the magnetic declination to be used by the alignment and fusion processing
```

Comment:
Suggest

// Return the magnetic declination in radians to be used by the alignment and fusion processing.